package.xml

```diff
@@ -32,8 +32,8 @@
   <depend>tf2_geometry_msgs</depend>
   <depend>tf2_ros</depend>
   <depend>yaml-cpp</depend>
+  <depend>geographiclib</depend>
```

Comment:
The rest of this list seems sorted :slightly_smiling_face:

Reply:
FIxed